src/script/auth/page/Index.tsx

```diff
@@ -67,16 +66,13 @@ const IndexComponent = ({defaultSSOCode}: Props & ConnectedProps & DispatchProps
   return (
     <Page>
       <ContainerXS centerText verticalCenter style={{width: '380px'}}>
-        <SVGIcon
+        <LogoFullIcon
           aria-hidden="true"
-          scale={1.3}
-          realWidth={78}
-          realHeight={25}
+          width={102}
+          height={33}
```

Comment:
Here we apply the `scale` factor directly (that's why the height/width values are different)